chevah/python-package
# [#5203] Cython addition.

Scope
=====

see if it is easy to get cython for our Linux and Windows python-package version

MacOS is optional.


Changes
=======

This was completely problem-free for all the platforms.

On Windows I have used the upstream wheel, which according to their documentation requires using Visual C on Windows.

**Drive-by fixes**:
  * updated SQLite for latest security issues: https://blade.tencent.com/magellan/index_en.html
  * detect MSYS env under Windows.

How to try and test the changes
===============================

reviewers: @adiroiban 

Check changes, eg. with:
```
git diff master paver.conf chevah_build python-modules/chevah-python-test/ paver.sh
```

Run the automated tests.
